scala/scala.bzl

```diff
@@ -164,6 +187,17 @@ def _write_test_launcher(ctx, jars):
       output=ctx.outputs.executable,
       content=content)
 
+def _collect_srcjars(targets):
+  srcjars = set()
+  for target in targets:
+    print(target) #TODO remove
+    #TODO perhaps we want to unify the naming?
+    if hasattr(target, "srcjar"):
```

Comment:
I see, you use it below in the rule that generates the scala, so you will need to keep `srcjar` but I don't think we need `srcjars` below, do we?